src/main/java/wtf/choco/arrows/AlchemicalArrows.java

```diff
@@ -122,6 +122,7 @@ public void onEnable() {
         // Register alchemical arrows
         this.getLogger().info("Registering default alchemical arrows and their recipes");
         this.createArrow(new AlchemicalArrowAir(this), "Air", Material.FEATHER); // Custom model data 132 (+1 for every registration)
+        this.createArrow(new AlchemicalArrowChain(this), "Chain", Material.FIREWORK_ROCKET);
```

Comment:
Additional note I wanted to make, might there be a better item for chain arrows? I'm not sure firework rockets make a whole lot of sense here. Certainly open to accepting firework rockets as an option, just wondering if there may be an alternative.

Reply:
Open for suggestions here. I was trying to find a balance between something that made sense (firework explodes into lots of other projectiles) but not too easy to obtain.

Reply:
And that's my concern as well. The process it takes to obtain this item seems slightly too demanding in my opinion. Perhaps a slimeball is most appropriate here.

Reply:
That works too. I'll update and change colors to match.